src/drive/web/modules/filelist/File.jsx

```diff
@@ -321,7 +320,6 @@ const File = props => {
       >
         <FileThumbnail
           file={attributes}
-          withSharedBadge={withSharedBadge}
```

Comment:
This is not an `unused props`  as described in the commit, but more an always true prop now, right?

Reply:
You're right, I'll edit the commit title